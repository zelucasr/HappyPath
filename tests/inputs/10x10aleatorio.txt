10 10
0 99
5 12 15 17 20 21 24 27 29 33 36 39 42 45 48 51 54 57 63 66 70 71 72 75 78 84 87 96
