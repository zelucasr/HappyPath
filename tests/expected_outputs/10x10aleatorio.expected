
0 to 99: 0 0 1 2 3 13 23 22 32 31 41 40 50 60 61 62 52 53 43 44 34 35 25 26 16 6 7 8 18 28 38 37 47 46 56 55 65 64 74 73 83 93 94 95 85 86 76 77 67 68 69 79 89 99 